README.md

```diff
@@ -434,7 +455,104 @@ const sdk = await init();
 sdk.applicationNavigator.openContentItem({ id: 'CONTENT_ITEM_ID...' });
 ```
 
-# Client (Supported on Content Field & Dashboard extensions)
+# Content Editor
+
+### Get Current Model
+
+Fetching will give you the current model of the content item you're viewing.
+
+```js
+const sdk = await init();
+const model = await sdk.form.getValue();
+```
+
+### Set Model
+
+Setting the model will return Error Report if there is any invalid content
+
+```js
+const sdk = await init();
+try {
+  await sdk.form.setValue({ title: "hello world", })
+} catch (errors) {
+  // retuens error report 
+}
+
+if (errors.length) {
+  console.log(errors);
+}
+```
+
+### Validate
+
+You can validate an object against the schema and get the full error report back
+
+```js
+const sdk = await init();
+const errors = await sdk.form.validate({
+  title: 'hello world',
+});
+
+if (errors) {
+  console.log(errors);
+}
+
+//
+[
+  {
+    errorCategory: 'DATA',
+    errorData: {
+      keyword: 'maxLength',
+      params: {
+        limit: 10,
+      },
+    },
+    errorSeverity: 'FATAL',
+    errorType: 'DATA_VALIDATION_ERROR',
+    message: 'should NOT be longer than 10 characters',
+    schema: {
+      id: 'https://simple-text.com',
+      originalId: 'https://simple-text.com',
+      location: {
+        pointer: '/properties/title/maxLength',
+      },
+    },
+    data: {
+      location: {
+        pointer: '/title',
+      },
+    },
+  },
+];
+```
+
+### isValid
+
+You can get a boolean value whether a piece of content is valid
```

Comment:
above seems like it could do with rephrasing to me. Maybe `A boolean value is returned showing whether the content is valid` or ` Returns true if valid`?